### PR TITLE
Return certificate and accept token in client

### PR DESCRIPTION
**Pull request by avin101**
- Updated client to return "Certificate" in the response of sign configuration privileges.
- Overloaded client constructor to accept bearer token.

### CODE REVIEW

**Review by smahapatra86 — APPROVED** [review]

File: src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs [hunk 1]
```diff
@@ -19,20 +19,40 @@ public sealed class FileSecurityClient : IDisposable
...
             this.options = options ?? throw new ArgumentNullException(nameof(options));
             this.tokenProviderFactory = tokenProviderFactory ?? throw new ArgumentNullException(nameof(tokenProviderFactory));
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FileSecurityClient"/> class using bearer token.
+        /// </summary>
+        /// <param name="httpClient">The HTTP client to use. The caller is expected to manage this resource and it will not be disposed.</param>
+        /// <param name="options">The required configuration options.</param>
+        /// <param name="bearerToken">This is required to get certificate from File Security.</param>
```
Comment (thecorrado) [inline src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs]: I would slight change description of that field:
_Required access token to authenticate with File Security module_

File: src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs [hunk 2]
```diff
@@ -335,12 +355,23 @@ internal InternalClient CreateClient()
...
-            this.internalClient = new InternalClient(new TokenCredentials(tokenProvider))
+            TokenCredentials credentials;
+            if (!string.IsNullOrEmpty(this.bearerToken))
             {
-                BaseUri = this.options.FileSecurityServiceUri,
-            };
+                credentials = new TokenCredentials(this.bearerToken);
+                this.internalClient = new InternalClient(credentials)
+                {
+                    BaseUri = this.options.FileSecurityServiceUri,
```
Comment (thecorrado) [inline src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs:364]: [minor] comma at end is not needed

**Review by thecorrado — CHANGES_REQUESTED** [review]

File: src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs (hunk 2 above)
Comment (avin101) [reply src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs:364]: Stylecop analyzer is prompting for a trailing comma. So looks like that is needed.

File: src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs (hunk 1 above)
Comment (avin101) [reply src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs]: updated as suggested.

**Review by thecorrado — APPROVED** [review]

**Pull request merged**